src/odemis/acq/align/test/delphi_test.py

```diff
@@ -104,7 +104,13 @@ def test_find_hole_center(self):
         data[0].shape = Y, X
 
         hole_coordinates = delphi.FindCircleCenter(data[0], 0.02032, 6, darkest=True)
-        expected_coordinates = (0.0052705, -0.0018415)  # (391.5, 257.5) px
+
+        # depending on the cv2 version which is related to the Ubuntu version the expected outcome is different
+        if delphi.cv2.__version__[:3] == "3.2":
```

Comment:
a bit nit-picky but this will also return True for version 3.21, you can do something like:
```
if ".".join(delphi.cv2.__version__.split(".")[:2]) == "3.2"
```